test/services/supplementary-billing/fetch-charge-versions.service.test.js

```diff
@@ -41,19 +41,21 @@ describe('Fetch Charge Versions service', () => {
     let chargeElement
     let chargePurpose
     let changeReason
+    let licence
 
     beforeEach(async () => {
       billingPeriod = {
         startDate: new Date('2022-04-01'),
         endDate: new Date('2023-03-31')
       }
 
-      const { licenceId } = await LicenceHelper.add({
+      licence = await LicenceHelper.add({
         regionId,
         isWaterUndertaker: true,
         includeInSrocSupplementaryBilling: true,
         includeInSupplementaryBilling: 'yes'
       })
+      const licenceId = licence.licenceId
```

Comment:
We could destructure this if we wanted.
```suggestion
      const { licenceId } = licence
```

Reply:
💯 agree. Good spot!